config.json

```
@@ -1,3 +1,3 @@
 {
-  "API_KEY": ""
-}
+  "API_KEY": "8e6f6608e3154b1a956193827cdaec4c"
```

You may want to change this now that it is in the public domain. Also, please be careful and not push keys to Github.

---

```suggestion
  "API_KEY": "8e6f6608e3154b1a956193827cdaec4c"
```
Thanks, removed it.

Also, I am getting a lot of linting errors, any idea how to resolve those?
Currently using VS Code, on Windows 10.

---

Thanks, removed it.

Also, I am getting a lot of linting errors, any idea how to resolve those?
Currently using VS Code, on Windows 10.